src/BaseModel.php

```diff
@@ -41,6 +43,12 @@ class BaseModel
         "https://openactive.io/ns-beta"
```

Comment:
should this be slightly user configurable?

Reply:
As agreed, will leave this for now.